service/debugger/locations.go

```diff
@@ -0,0 +1,291 @@
+package debugger
+
+import (
+	"debug/gosym"
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/derekparker/delve/service/api"
+)
+
+const maxFindLocationCandidates = 5
```

Comment:
It's not strictly necessary, I wanted to include in the error message some extra information about the source for the ambiguity but I also didn't want the error message to potentially be very long.